### PR TITLE
Michiel/Change build to exclude unsafe script

### PR DESCRIPTION
**Pull request by michielmulders**
**What did I do?**

- Used the `--no-unsafe-inline` flag for `vue-cli` which allows to create a build avoiding inline scripts.
- However, sometimes there are some scripts that just can't be locked down. Here, we can use a hash that represents the inline script as a second layer measure. (more info about this [here](https://www.troyhunt.com/locking-down-your-website-scripts-with-csp-hashes-nonces-and-report-uri/)).

**State?** Ready for review/merge.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Lunie is the staking and governance platform for proof-of-stake blockchains.",
   "author": "Lunie International Software Systems Inc. <hello@lunie.io>",
   "scripts": {
-    "build": "node tasks/create-version-file.js && vue-cli-service build --skip-plugins pwa",
+    "build": "node tasks/create-version-file.js && vue-cli-service build --no-unsafe-inline --skip-plugins pwa",
```
Comment (faboweb) [inline package.json:7]: nice

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
     "buildsentry": "node tasks/create-version-file.js && vue-cli-service build --skip-plugins pwa && chmod 755 tasks/sentry.sh && ./tasks/sentry.sh",
```
Comment (faboweb) [inline package.json:8]:
```suggestion
    "buildsentry": "node tasks/create-version-file.js && vue-cli-service build --skip-plugins pwa --no-unsafe-inline && chmod 755 tasks/sentry.sh && ./tasks/sentry.sh",
```

**Comment by faboweb** [comment]
Still shows errors

File: package.json (hunk 2 above)
Comment (michielmulders) [reply package.json:8]: Would swap the order here!

**Comment by michielmulders** [comment]
@faboweb Hash added for the two remaining unsafe ones :)
Good to go: https://deploy-preview-3923--lunieio.netlify.app/cosmos-hub/portfolio

File: netlify.toml [hunk 3]
```diff
@@ -11,7 +11,7 @@
       default-src 'self' https://*.lunie.io https://*.netlify.app https://www.google-analytics.com https://monitoring.lunie.io:9000 https://js.intercomcdn.com;
       connect-src 'self' https://api-iam.intercom.io https://widget.intercom.io wss://*.intercom.io wss://production-api.lunie.io/graphql https://production-api.lunie.io wss://staging-api.lunie.io/graphql https://staging-api.lunie.io https://monitoring.lunie.io:9000;
       img-src 'self' data: https://*.lunie.io https://*.netlify.app https://www.google-analytics.com https://s3.amazonaws.com https://lunie.fra1.digitaloceanspaces.com;
-      script-src 'self' 'unsafe-inline' 'unsafe-eval' https://*.lunie.io https://*.netlify.app https://www.google-analytics.com https://api-iam.intercom.io https://js.intercomcdn.com https://widget.intercom.io;
+      script-src 'self' 'unsafe-eval' https://*.lunie.io https://*.netlify.app https://www.google-analytics.com https://api-iam.intercom.io https://js.intercomcdn.com https://widget.intercom.io 'sha256-G4vvwkai1flA+XSDCXPR0KDlLS9x0zsszpJIV6tiApA=' 'sha256-FakRMA+u6is/Tx11H/OX3XBouQxbbE0TmeFHUdeXZ5o=';
```
Comment (faboweb) [inline netlify.toml:14]: what are those hashes?

Comment (faboweb) [reply netlify.toml:14]: where do those hashes come from?

**Pull request merged**